docusaurus/docs/learn/quickstarts/network/help/upgrade-quickstart-network.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Upgrade a Quickstart Network
+id: upgrade-quickstart-network
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This document explains the process for backing up, and upgrading your Ziti Controller and Ziti Router(s).
+
+## Backup Existing Network
+You can certainly back up the entire Ziti Network directory (defaulted to `~/.ziti`). However, if you prefer to only 
```

Comment:
> You can certainly back up the entire Ziti Network directory

Maybe change to: "If you followed the ${quickstart}, you can backup your entire network quickly and easily by copying the entire $ZITI_HOME directory (defaulted to `$HOME/quickstart/$(hostname -s)/$(hostname -s)`). If you prefer to backup the minimum files necessary....." If you change this to work with the type of quickstart, then the 'if you followed' part wouldn't be needed (of course)